webviz_subsurface/plugins/_co2_leakage/views/mainview/mainview.py

```diff
@@ -0,0 +1,185 @@
+        }
+        CONTENT_PARENT = {
+            "flex": 3,
+            "display": "flex",
+            "flexDirection": "column",
+        }
+
+    def inner_layout(self) -> Component:
+        return html.Div(
+            [
+                wcc.Frame(
+                    # id=self.register_component_unique_id(LayoutElements.MAP_VIEW),
+                    color="white",
+                    highlight=False,
+                    children=[
+                        html.Div(
+                            [
+                                DeckGLMap(
+                                    id=self.register_component_unique_id(self.Ids.DECKGL_MAP),
+                                    layers=[],
+                                    bounds=INITIAL_BOUNDS,
+                                    coords={"visible": True},
+                                    scale={"visible": True},
+                                    toolbar={"visible": True},
+                                    coordinateUnit="m",
+                                    colorTables=_color_tables(),
+                                    zoom=-7,
+                                ),
+                            ],
+                            style=self.Style.MAP_WRAPPER,
+                        ),
+                        html.Div(
+                            wcc.Slider(
+                                id=self.register_component_unique_id(self.Ids.DATE_SLIDER),
+                                step=None,
+                                marks={0: ''},
+                                value=0,
+                            ),
+                            id=self.register_component_unique_id(self.Ids.DATE_WRAPPER),
+                        ),
+                    ],
+                    style=self.Style.MAP_VIEW,
+                ),
+                wcc.Frame(
+                    # id=get_uuid(LayoutElements.PLOT_VIEW),
+                    style=self.Style.PLOT_VIEW,
+                    children=SummaryGraphLayout(
+                        self.register_component_unique_id(self.Ids.BAR_PLOT),
+                        self.register_component_unique_id(self.Ids.TIME_PLOT),
+                    ).children
+                )
+            ],
+            style=self.Style.CONTENT_PARENT
+        )
+
+
+class SummaryGraphLayout(html.Div):
+    def __init__(self, bar_plot_id, time_plot_id, **kwargs):
+        super().__init__(
+            children=[
+                dcc.Graph(
+                    id=bar_plot_id,
+                    figure=go.Figure(),
+                    config={
+                        "displayModeBar": False,
+                    }
+                ),
+                dcc.Graph(
+                    id=time_plot_id,
+                    figure=go.Figure(),
+                    config={
+                        "displayModeBar": False,
+                    }
+                ),
+            ],
+            **kwargs,
+        )
+
+
+def _color_tables():
```

Comment:
This is defined in two files. Suggest moving to a separate file.

Reply:
Well picked up. Fixed as suggested